src/components/calcite-button-with-dropdown/readme.md

```diff
@@ -0,0 +1,50 @@
+# calcite-button-with-dropdown
```

Comment:
Can we add a description of what this component is for here? I'm not totally sure myself, so would be appreciated.